pom.xml

```diff
@@ -200,7 +200,7 @@
     <dependency>
       <groupId>com.github.tomakehurst</groupId>
       <artifactId>wiremock</artifactId>
-      <version>2.27.2</version>
+      <version>3.0.1</version>
       <scope>test</scope>
```

Comment:
Overall, it is good idea to move as version tag
e.g `<aspectj.version>1.9.21.1</aspectj.version>`